rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -160,36 +159,36 @@ def __init__(
         # declare class instance for CountVectorizer
         self.vectorizer = vectorizer
 
-    def _tokenizer(self, text):
-        """Override tokenizer in CountVectorizer."""
-
-        text = re.sub(r"\b[0-9]+\b", "__NUMBER__", text)
+    def _get_message_text(self, message):
+        if message.get("spacy_doc"):  # if lemmatize is possible
+            text_tokens = [t.lemma_ for t in message.get("spacy_doc")]
+            # text = " ".join([t.lemma_ for t in message.get("spacy_doc")])
+        elif message.get("tokens"):  # if directly tokens is provided
```

Comment:
should it rather be the other way around? preferring tokens over spacy?

Reply:
hm, I think we assume that spacy lemma is better than using raw word tokens

Reply:
:+1: